public/app/features/alerting/unified/utils/rule-id.ts

```diff
@@ -173,10 +173,10 @@ function hash(value: string): number {
   return hash;
 }
 
-// this is used to identify lotex rules, as they do not have a unique identifier
-function hashRulerRule(rule: RulerRuleDTO): number {
+// this is used to identify rules, mimir / loki rules do not have a unique identifier
+export function hashRulerRule(rule: RulerRuleDTO): string {
```

Comment:
do we want a test for this function?

Reply:
Yea, I think we most certainly do – will append one